sdk/keyvault/keyvault-keys/CHANGELOG.md

```diff
@@ -9,6 +9,7 @@
   - If a traced operation throws an exception we will now properly record the exception message in the tracing span.
   - Finally, naming conventions have been standardized across the KeyVault libraries taking the format of `Azure.KeyVault.<PACKAGE NAME>.<CLIENT NAME>`.
 - Fixed an issue where retrying a failed initial Key Vault request may result in an empty body.
+- [Breaking] Removed the now unused `LocalCryptographyAlgorithmName` type.
```

Comment:
I think we should mention when this was used before, like "this was used by X, but X got removed on Y version". Something like that?